src/window.py

```diff
@@ -579,7 +580,7 @@ def load_stl(self, stl_actor):
 
         self.render.AddActor(self.stlActor)
         self.state_stl()
-        self.render.ResetCamera()
+        #self.render.ResetCamera()
```

Comment:
Is it possible to comment out this line?

Reply:
If you tried and it worked better, I trust you)

Reply:
Ok. You can test on your latest build. Move the model out of the camera and click colorize or slice. The camera will move to the model for a second and then back.